src/transform/src/dataflow.rs

```diff
@@ -185,11 +185,21 @@ fn optimize_dataflow_demand(dataflow: &mut DataflowDesc) {
             .extend(0..dataflow.arity_of(&input_id));
     }
 
-    // Propagate demand information from outputs to inputs.
```

Comment:
I'm not allowed to add a comment there, but the doccomment for the method should be updated.